packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/utils/getActionTypesDefaultMessages.js

```diff
@@ -0,0 +1,86 @@
+const actionTypes = {
+  ENTRY_CREATE: 'entry.create',
+  ENTRY_UPDATE: 'entry.update',
```

Comment:
I'm not sure this object is necessary, couldn't you just use the values directly in the switch? Since it's only used there

Reply:
I think the split doesn't work every time, for example the login event is a bit different, and we need custom formatting for some things (e.g. 'content-type' -> 'Content type').

Of course we could try to further manipulate the string, but I'm not super comfortable with it. Since there are not that many events I think it's a good use case for a "dumb" switch case